sqlmesh/cli/main.py

```diff
@@ -1219,3 +1279,115 @@ def state_import(obj: Context, input_file: Path, replace: bool, no_confirm: bool
     """Import a state export file back into the state database"""
     confirm = not no_confirm
     obj.import_state(input_file=input_file, clear=replace, confirm=confirm)
+
+
+def _interactive_init(
```

Comment:
Nit: I feel like these should go to `example_project.py`, so that we only have CLI command entrypoints in this file.